src/python/pants/pantsd/pants_daemon.py

```diff
@@ -325,7 +325,9 @@ def run_sync(self):
                 time.sleep(self.JOIN_TIMEOUT_SECONDS)
 
             # We're exiting: join the server to avoid interrupting ongoing runs.
-            # TODO: This will happen via #8200.
+            self._logger.info("waiting for ongoing runs to complete before exiting...")
```

Comment:
Nit: I personally think capitalizing INFO level logs reads more professionally. (I know the recent PRs are using lowercase.)